scripts/prepare_transport_data.py

```diff
@@ -138,23 +138,23 @@ def prepare_transport_data(n):
 
     # divide out the heating/cooling demand from ICE totals
     # and multiply back in the heating/cooling demand for EVs
-    ice_correction = (transport_shape * (1 + dd_ICE)).sum() / transport_shape.sum()
-
-    if snakemake.config["custom_data"]["transport_demand"]:
-        energy_totals_transport = nodal_energy_totals["total road"]
-
-        transport = transport_shape.multiply(energy_totals_transport) * 1e6 * Nyears
-    else:
-        energy_totals_transport = (
-            nodal_energy_totals["total road"]
-            + nodal_energy_totals["total rail"]
-            - nodal_energy_totals["electricity rail"]
-        )
-        transport = (
-            (transport_shape.multiply(energy_totals_transport) * 1e6 * Nyears)
-            .divide(efficiency_gain * ice_correction)
-            .multiply(1 + dd_EV)
-        )
+    ice_correction = (transport_shape * (1 + dd_ICE)).sum() / transport_shape.sum() # currently not used same for dd_EV
+
+    # if snakemake.config["custom_data"]["transport_demand"]:
+    energy_totals_transport = nodal_energy_totals["total road"]
+
+    transport = transport_shape.multiply(energy_totals_transport) * 1e6 * Nyears
+    # else:
+    #     energy_totals_transport = (
+    #         nodal_energy_totals["total road"]
+    #         + nodal_energy_totals["total rail"]
+    #         - nodal_energy_totals["electricity rail"]
```

Comment:
I am not familiar with the model changes of the last months, but as far as I can remember road and rail transport were summed up, except from the electricity demand for rail transport which was added as load on the AC buses.